spec/spec_helper.rb

```diff
@@ -53,7 +53,7 @@
   #   # is tagged with `:focus`, all examples get run. RSpec also provides
   #   # aliases for `it`, `describe`, and `context` that include `:focus`
   #   # metadata: `fit`, `fdescribe` and `fcontext`, respectively.
-  #   config.filter_run_when_matching :focus
+  config.filter_run_when_matching :focus
```

Comment:
Nice to have enabled in every repo so you can just prepend any `it` or `context` with `f` to run just that particular set of tests.